geonode/layers/templates/layers/layer_leaflet_map.html

```diff
@@ -190,19 +190,12 @@
             zoom_to_box(map, [{{ resource.bbox_string }}]);
         {% endif %}
 
+        var tile_layer;
         {% if resource.get_tiles_url %}
-
             tile_layer = L.tileLayer("{{ resource.get_tiles_url|safe }}",
                     {
                         'opacity': 0.8
                     });
-
-            L.tileLayer.betterWms('{{ resource.ows_url|safe }}', {
-              layers: '{{ resource.typename }}',
-              transparent: true,
-              format: 'image/png'
-            }).addTo(map);
```

Comment:
Previously, this block doesn't have an if clause, probably a mistake. So, both tile url and ows url gets rendered.